src/com/seafile/seadroid2/ui/fragment/ReposFragment.java

```diff
@@ -450,6 +450,10 @@ private void resend() {
         // onPostExecute displays the results of the AsyncTask.
         @Override
         protected void onPostExecute(List<SeafRepo> rs) {
+            if (mActivity == null)
+                // this occurs if user navigation to another activity
+                return;
+
```

Comment:
This fix looks ok to me. But I'm thinking may be the best practice should be to cancel all async tasks when the activity/fragment is stopped, instead of checking the null-ness of the activity object.